pkg/aspect/registry.go

```diff
@@ -39,6 +39,6 @@ func Inventory() []APIBinding {
 		{NewListCheckerManager(), config.CheckMethod},
 		{NewLoggerManager(), config.ReportMethod},
 		{NewAccessLoggerManager(), config.ReportMethod},
-		{NewQuotaManager(), config.QuotaMethod},
+		{NewQuotaManager(), config.CheckMethod},
```

Comment:
should we add instead of replace? something like:

```
{NewQuotaManager(), config.QuotaMethod},
{NewQuotaManager(), config.CheckMethod},
```

I think that would allow us to test the transition.